docs/UserGuide.md

```diff
@@ -39,268 +39,292 @@ The program simulates a real life workflow:
+#### `personedit`: Edit an employee's information
 
 Edit the information of an existing employee. Use this in the event that an employee's details change.
 
-Rate updates will only take effect with jobs that are pending completion; payout amounts for already-completed jobs will not change.
+Rate updates will only take effect with jobs that are pending completion; payout amounts for already-completed (i.e. marked with `mark`) jobs will not change.
 
 Format: `personedit PERSON_INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [r/RATE] [t/TAG]...​`
 
-Example: `personedit 2 p/62353535` edits the second person's phone number to 62353535
+Examples:
 
-`personedit 3 t/Hardware t/Network` edits the third person's tags to be `Hardware` and `Network` instead of the tags originally associated.
+* `personedit 2 p/62353535` changes the second person's phone number to 62353535.
+* `personedit 3 t/Hardware t/Network` changes the third person's tags to `Hardware` and `Network` instead of their original tags.
 
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Note:**<br>
 
-* Names containing `/` are not accepted. Use `-` instead of `/` if an employee's name has it. For example:
-  Use `Ravi s-o Veegan` instead of `Ravi s/o Veegan`.
+* Attributes containing `/` are not accepted. Replace `/` with another character (e.g. `-`) if any attribute contains `/`.
+
+  For example: Use `Ravi s-o Veegan` instead of `Ravi s/o Veegan`, and `3-5 Jalan Trus` instead of `3/5 Jalan Trus`.
 
 * The maximum value for the hourly rate of an employee is $1,000,000.
 
-* When editing tags, new tags will not be added to the existing tags. Rather, all existing tags will be
-replaced by new tags. Additionally, all tags can be cleared by typing `t/` without specifying any tags.
+* When editing tags, new tags will not be added to the existing tags. Instead, all existing tags will be replaced by new tags.
+
+* To clear all tags, add `t/` to the command without specifying any other tags.
   
 </div>
 
-#### How to delete an employee's information? `persondelete`
-Deletes the employee referred to by the index. This is irreversible. Removes the deleted employee from all associated jobs as well.<br>
+#### `persondelete`: Delete an employee
 
-❗️Caution: This is irreversible.
+Deletes the employee referred to by the index. This also removes the deleted employee from all associated jobs.<br>
+
+<div markdown="block" class="alert alert-warning">
+
+**:warning: Caution:**<br>
+
+This is irreversible.
+
+</div>
 
 Format: `persondelete PERSON_INDEX`
 
-Example: `persondelete 3` deletes the third person in the list
+Example: `persondelete 3` deletes the third person in the list.
 
-#### How to search for a person by name or tag? `personfind`
+#### `personfind`: Find employees by name or tag
 
-Finds all people by a certain name and/or tag. 
+Finds all employees that have the given keyword(s) in their names or tags, and lists them in the employee table.
```

Comment:
I used 'employee list' in the labelled UI screenshot above so they might be confused if it is called table now
```suggestion
Finds all employees that have the given keyword(s) in their names or tags, and lists them in the employee list.
```